### DISCUSSION TITLE
UniGetUI (formerly WingetUI) 3.1.0 alpha 0

### DISCUSSION THREAD

**Discussion by marticliment** [Devolutions/UniGetUI#2224] [Show and tell]
![](https://img.shields.io/github/downloads/marticliment/wingetui/3.1.0-alpha0/total?style=for-the-badge&color=red)

<b>IMPORTANT:
This is the first development release to implement the UniGetUI rebranding.
Instaling this release will automatically migrate settings and other data to new folders (WingetUI -> UniGetUI), so rolling back to older versions of WingetUI may cause settings to be reset.
</b>

## Changes
 - Rebranding from WingetUI to UniGetUI
   - Any occurrences of WingetUI have been changed to UniGetUI
   - Settings have been moved from user profile to AppData\Local
 - Improvements to the package managers
   - All dotnet-based package managers do call the nuget api when possible. This dramatically speeds up find packages and loading package details, as well as provides more reliability.
   - All dotnet-based packages will load the official package icon
   - WinGet implements the WinGet COM Api, for improved performance and reliability. WinGet packages do not show elypsis (`...`) anymore.
   - Microsoft Store packages (currently msix only) will show icons and screenshots
   - The icon caching mechanism has been rebuilt from the grounds up, providing more compatibility with Widgets for WingetUI
 - Improvements to Software pages 
 - Better search experience on the Discover Software page
 - Refactoring. Lots of refactoring
 - Other improvements and internal changes.

SHA256: `0ec58bc34ff63f0bc1904fe88e8cdd247a2348e259ce8394911979aef5477281`

<hr /><em>This discussion was created from the release <a href='https://github.com/marticliment/WingetUI/releases/tag/3.1.0-alpha0'>UniGetUI (formerly WingetUI) 3.1.0 alpha 0</a>.</em>

**Comment by Mertsch** [comment]
Wow, this is amazing! Keep up the good work.

**Comment by Jonathing** [comment]
Loving the work being put into this project. Keep on rocking.

I kind of think UniGet would be a better name than UniGetUI since the "UI" part was born from this program being a UI for Winget. Since it is more of its own thing now, I think it's a little redundant.

**Reply by aggietallboy** [reply]
Thank you for the new update!! Installing and playing with it now.

Did you get the load package from command line capability added in this release?

**Reply by marticliment** [reply]
No. While the respective Feature Request is open the feature has not been implemented yet

**Comment by ipisano** [comment]
Can anyone check if scoop works here? I can't get it to work with latest stable, it's installed in my system and works from terminal but doesn't get recognized by WingetUI. (I already posted about this in the relevant issue thread)

**Reply by Schielegerhard** [reply]
Scoop works, but I have to deactivate and reactivate it manually in the settings every time I start UniGetUI. Only then is Scoop recognized. The same applies to npm.

**Comment by J-HS-L** [comment]
Compared to the stable release, how flaky is this. 
Is it still relatively stable/safe?  TY/BR!

**Reply by marticliment** [reply]
It depends. It might work 100% or it might always crash. This is why it is an alpha

**Reply by MAX4ARCH** [reply]
Works great, the only issue I got - it doesnt see my chocolatey packages installed.

**Reply by marticliment** [reply]
Please check this two folders:
 - `%userprofile%\AppData\Local\Programs\WingetUI\choco-cli\`
 - `%userprofile%\AppData\Local\UniGetUI\Chocolatey\`

Are there any contents there?

**Reply by MAX4ARCH** [reply]
> Please check this two folders:
> 
> * `%userprofile%\AppData\Local\Programs\WingetUI\choco-cli\`
> * `%userprofile%\AppData\Local\UniGetUI\Chocolatey\`
> 
> Are there any contents there?

Yes, I even tried to delete them, because I use system chocolatey. Reverted to stable for myself meanwhile. Again, thank you for your great effort and fast replies!)

**Reply by MAX4ARCH** [reply]
finally found what cause that issue, choco was only in administrator path, when I added to user path, everything become to be shown in unigetui. (v 3.1.0)

**Comment by Janzzze** [comment]
Found a couple of places where it's not rebranded to UniGetUI

<details>
<summary>Anywhere Windows displays the name its still WingetUI</summary>

![Screenshot 2024-06-01 194730](https://github.com/marticliment/WingetUI/assets/6020130/5ae117cc-a02f-4118-93a6-460e1c3ad31f)

![Screenshot 2024-06-01 162219](https://github.com/marticliment/WingetUI/assets/6020130/541945f3-3e9a-47d5-b8ce-4f1381d5a788)

![Screenshot 2024-06-01 192735](https://github.com/marticliment/WingetUI/assets/6020130/80f8fa15-7ed8-4b57-a10b-4867da35802a)

![Screenshot 2024-06-01 195434](https://github.com/marticliment/WingetUI/assets/6020130/988f5428-f22b-4984-8b60-d32c79dd2bfb)

</details>

<details>
<summary>The executable is still WingetUI.exe</summary>

![Screenshot 2024-06-01 193059](https://github.com/marticliment/WingetUI/assets/6020130/7798e838-b765-4239-8374-983250d8ab70)

![Screenshot 2024-06-01 192757](https://github.com/marticliment/WingetUI/assets/6020130/eae1ccc8-6a50-4423-af99-003dae575c56)

</details>

**Reply by marticliment** [reply]
Thanks for the report.

Please note that this is still an alpha release
